src/lotus_json/mod.rs

```diff
@@ -320,6 +320,40 @@ pub mod hexify_bytes {
     }
 }
 
+pub mod hexify_vec_bytes {
```

Comment:
Isn't https://docs.rs/hex/latest/hex/ doing it already?

Reply:
E.g., https://github.com/ChainSafe/forest/blob/ccfcf4dab725bbfce8fe7d1a373e307c1b9fa149/src/blocks/tests/serialization_vectors.rs#L84-L85

Or https://github.com/ChainSafe/forest/blob/18c5f2617369942c5157223dc0bee5a7331a61e9/src/lotus_json/mod.rs#L297-L321 (not sure if's not redundant as well)

Reply:
https://docs.rs/serde_with/latest/serde_with/hex/struct.Hex.html
Looks like they don't prepend with `0x` (if I'm looking at the right doc). Could we tweak that?

Reply:
`hexify_bytes` works only on fixed-size arrays, AFAIK.

Reply:
Maybe `with_prefix!` could help us for `0x`.

Reply:
This could help, perhaps. https://docs.rs/serde_with/latest/serde_with/macro.with_prefix.html